pandas/tests/io/test_excel.py

```diff
@@ -660,19 +606,17 @@ def test_read_from_py_localpath(self, ext):
         # GH12655
         from py.path import local as LocalPath
 
-        str_path = os.path.join(self.dirpath, 'test1' + ext)
+        str_path = os.path.join('test1' + ext)
         expected = read_excel(str_path, 'Sheet1', index_col=0)
 
-        abs_dir = os.path.abspath(self.dirpath)
-        path_obj = LocalPath(abs_dir).join('test1' + ext)
+        path_obj = LocalPath().join('test1' + ext)
```

Comment:
Given the fixture changes to the data directory, this invocation without an argument resolves to there automatically without need to resolve to an absolute path